packages/gestalt/src/ExpandableModuleBase.js

```diff
@@ -0,0 +1,108 @@
+// @flow strict
+import React, { type Node } from 'react';
+import Box from './Box.js';
+import IconButton from './IconButton.js';
+import Icon from './Icon.js';
+import TapArea from './TapArea.js';
+import Text from './Text.js';
+
+type Props = {|
+  title: string,
+  icon?: string,
```

Comment:
@galaxy0101 let's make sure we can only use Gestalt icons (see `Icon.js`: `icon?: $Keys<typeof icons>`)